crates/matrix-sdk/src/test_utils/mocks/mod.rs

```diff
@@ -1016,6 +1016,13 @@ impl MatrixMockServer {
         let mock = Mock::given(method("GET"));
         self.mock_endpoint(mock, RoomRelationsEndpoint::default()).expect_default_access_token()
     }
+
+    /// Create a prebuilt mock for the endpoint used to get the global account
+    /// data.
+    pub fn mock_global_account_data(&self) -> MockEndpoint<'_, GlobalAccountDataEndpoint> {
```

Comment:
I like to add examples to these, because it's not infrequent that people ask how these can be used.

Let's add one here as well.